subresource-loading.bs

```diff
@@ -171,24 +180,26 @@ changes:
        specific requirements for the error handling here, so currently an
        `error` event is fired similarly to the case of an empty `src` attribute.
 
-- Insert the following case to <a spec="html">prepare a script</a> step 27.2:
+- Insert the following case to <a spec="html">prepare the script element</a>
+  step 27.2:
```

Comment:
These step numbers need to be updated (again), but that could be a separate PR.

Reply:
Thanks. Yeah, step numbers can be updated later. Step numbers become out-of-sync easily. :(